packages/fuselage/src/components/Table/index.d.ts

```diff
@@ -19,6 +20,7 @@ type TableCellProps = ComponentProps<typeof Box> & {
 export const Table: ForwardRefExoticComponent<TableProps> & {
   Head: ForwardRefExoticComponent<TableHeadProps>;
   Body: ForwardRefExoticComponent<TableBodyProps>;
+  Foot: ForwardRefExoticComponent<TableFootProps>;
```

Comment:
`Footer` isn't a better name here?

Reply:
Maybe. I believe we have just kept `Foot` because `<tfoot />` exists on HTML5. We can rename it (as well as `Head` to `Header`) in the future, but keeping `Foot` (and `Head`) as an alias with some kind of deprecation notice.

Reply:
Makes total sense!